kubescape-operator/application.yaml

```diff
@@ -0,0 +1,26 @@
+---
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: kubescape-operator
+  namespace: argocd
+  annotations:
+    argocd.argoproj.io/sync-wave: '10'
```

Comment:
Please remove the sync wave, otherwise the external secret is going to complain because of the namespace

Reply:
Didn't we say we will remove the external secret anyway?